### PR TITLE
Function to configure AWS CLI

### PR DESCRIPTION
**Pull request by akaranjkar-qu**

### CODE REVIEW

File: misc/awscli.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                return 1
+                ;;
+        esac
+    done
+
+    if [[ ! -x "$(command -v aws)" ]]; then
+        echo "AWS CLI was not found.."
+        return 1
+    fi
+    if [[ ! -z ${REGION} ]]; then
```
Comment (hiyer) [inline misc/awscli.sh]: Either region should be required or we should use a default.

File: misc/awscli.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        aws configure set region ${REGION} --profile ${PROFILE}
```
Comment (hiyer) [inline misc/awscli.sh]: Is this the only configuration required if the cluster is role-based?

**Review by hiyer — CHANGES_REQUESTED** [review]

File: misc/awscli.sh (hunk 2 above)
Comment (akaranjkar-qu) [reply misc/awscli.sh]: Yes setting the region is sufficient

File: misc/awscli.sh (hunk 1 above)
Comment (akaranjkar-qu) [reply misc/awscli.sh]: Defaulted it to `us-east-1`

**Pull request merged**